apps/cow-fi/pages/careers/index.tsx

```diff
@@ -184,13 +188,25 @@ export default function Page({ siteConfigData, jobsData }: PageProps) {
 
 export const getStaticProps: GetStaticProps<PageProps> = async () => {
   const siteConfigData = CONFIG
-  const jobsData = await getJobs()
-
-  return {
-    props: {
-      siteConfigData,
-      jobsData,
-    },
-    revalidate: DATA_CACHE_TIME_SECONDS,
+  console.log('Fetching jobs data...')
+  try {
+    const jobsData = (await getJobs()) || {}
+    console.log('Jobs data fetched:', jobsData)
```

Comment:
I don't think we should have all of these console logs in the final version. Should we?